e2e/Makefile

```diff
@@ -39,7 +39,20 @@ clean:
 # This target can be used to delete namespaces that were created during testing. If a namespace is provided the user and
 # the namespace (as a prefix) must match.
 destroy-my-namespaces:
-	./remove_namespaces ${TEST_USERNAME} ${NAMESPACE}
+	./scripts/remove_namespaces ${TEST_USERNAME} ${NAMESPACE}
+
+# This target can be used to create a kind cluster that can be used for e2e testing.
+kind-setup:
+	@CHAOS_NAMESPACE=$(CHAOS_NAMESPACE) FDB_VERSION=$(FDB_VERSION) REGISTRY=$(REGISTRY) $(shell pwd)/scripts/setup_e2e.sh
```

Comment:
setup_e2e.sh creates an HA cluster by default. 
Good to have a setup to create non-HA cluster, which uses less resource and allows us to cover non-HA scenario.

Reply:
This calls the `./e2e/scrips/setup_e2e.sh` not the other script (`./scripts/setup_e2e.sh`) (I will rename that and move it into the `./e2e/scripts/` folder).